cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -117,6 +117,15 @@ static const char *nvidia_globs[] = {
 	"libnvidia-rtcore.so*",
 	"libnvidia-tls.so*",
 	"libnvoptix.so*",
+	"libnvdc.so*",
+	"libnvos.so*",
+	"libnvrm_gpu.so*",
+	"libnvimp.so*",
+	"libnvrm.so*",
+	"libnvrm_graphics.so*",
+	"libnpp{c,ig,ial,icc,idei,ist,if,im,itc}*.so*",
```

Comment:
Also looking at the existing lines, is this regex acceptable here or shall I write each so file name individually ?

Reply:
Those are simple globs, you need to list each individually. While at it, please move the libnvidia-tls and libvdapu_nvidia above tegra entries i.e.:
```
"tls/libnvidia-tls.so*",
"vdpau/libvdpau_nvidia.so*",

// additional libraries for Tegra
"libnppc*.so*",
"libnppig*.so*",
"libnppial*.so*",
...
```

Reply:
Thanks for the comment. Pushed the requested change.